pkg/atf/constants.go

```diff
@@ -0,0 +1,23 @@
+package atf
+
+const providerStanza = `
+	provider hpegl {
+		vmaas {
+		}
+		caas {
+		}
+		metal {
+		}
+	}
+`
+
+var accTestPath = "../../acc-testcases"
```

Comment:
default path for acceptance test is fine since the test cases are stored in corresponding child repo, but once it is copied to parent library they should be in different directory ( to avoid file conflicts) and run acceptance test using `TF_ACC_TEST_PATH`